gravitee-apim-console-webui/src/management/api/analytics/pathMappings/api-path-mappings-add-dialog/api-path-mappings-add-dialog.component.html

```diff
@@ -0,0 +1,76 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<h2 mat-dialog-title class="title">Add Path Mapping</h2>
+
+<mat-dialog-content class="content">
+  <mat-tab-group dynamicHeight mat-stretch-tabs (selectedTabChange)="onSelectedTab($event)">
+    <!-- Add Path Mapping -->
+    <mat-tab [label]="tabLabels.AddPathMapping">
+      <div class="content__url-tab">
+        <mat-dialog-content class="content" [formGroup]="pathFormGroup">
+          <mat-form-field class="content__path__form-field" appearance="fill">
```

Comment:
<ma-error ...> are cropped.

![Screenshot 2023-01-20 at 14 56 04](https://user-images.githubusercontent.com/25704259/213714012-8d238e31-c96e-4a9b-9553-7b0bf9246e96.png)

I think we can avoid it by replacing the `padding: 0` by a   `padding-bottom: 4px;`

Reply:
This is something that I wanted to talk about it. Do we want to have the same checks for both pages or not? The old page doesn't check

Reply:
done